src/plugins/workspace/server/routes/index.ts

```diff
@@ -255,4 +258,19 @@ export function registerRoutes({
       return res.ok({ body: result });
     })
   );
+
+  router.get(
+    {
+      path: `${WORKSPACES_API_BASE_URL}/is_workspace_enabled`,
```

Comment:
Could we add a route path for workspace settings? We also need one API to check if the permission is enabled, want to call one API to return all the settings.

Reply:
I think so, let me modify this route to return all settings workspace has.

Reply:
Done for this